testing/infra/k8s/base/stack/apm-server.yaml

```diff
@@ -1,31 +1,33 @@
-apiVersion: agent.k8s.elastic.co/v1alpha1
-kind: Agent
+apiVersion: apm.k8s.elastic.co/v1
+kind: ApmServer
 metadata:
   name: apm-server
-  namespace: default
 spec:
   version: 8.8.0-ba3f07b2-SNAPSHOT
-  mode: fleet
-  policyID: eck-apm-server
-  kibanaRef:
-    name: kibana
-  fleetServerRef:
-    name: fleet-server
-  deployment:
-    replicas: 1
-    strategy:
-      # Can be changed to "Replace" for extremely memory limited environments.
-      type: "RollingUpdate"
-    podTemplate:
-      spec:
-        containers:
-        - name: agent
+  count: 1
+  http:
+    tls:
+      selfSignedCertificate:
+        disabled: true
+  config:
+    logging:
+      level: debug
+    output:
+      elasticsearch:
+        hosts: ["http://elasticsearch-es-http.default.svc:9200"]
```

Comment:
```suggestion
```

I was able to remove just this, and the server successfully connects to ES and reads from `.apm-agent-configuration`.